website/docs/reference/resource-configs/snowflake-configs.md

```diff
@@ -182,7 +182,43 @@ models:
 
 ## Configuring virtual warehouses
 
-The default warehouse that dbt uses can be configured in your [Profile](/reference/profiles.yml) for Snowflake connections. To override the warehouse that is used for specific models (or groups of models), use the `snowflake_warehouse` model configuration. This configuration can be used to specify a larger warehouse for certain models in order to control Snowflake costs and project build times.
+The default warehouse that dbt uses can be configured in your [Profile](/reference/profiles.yml) for Snowflake connections. To override the warehouse that is used for specific models (or groups of models), use the `snowflake_warehouse` model configuration. This configuration can be used to specify a larger warehouse for certain models in order to control Snowflake costs and project build times. The following config changes the warehouse for a single model with a config() block.
+  
+<File name='models/events/sessions.sql'>
```

Comment:
hey @patkearns10 , will approve this for you soon but wanted to see if i can create tabs for the SQL and yml examples so it requires less scrolling for the user experience. see this [slack thread](https://dbt-labs.slack.com/archives/C02NCQ9483C/p1669995969930599)

Reply:
ok got it!

`<Tabs
  defaultValue="dbt_project.yml"
  values={[
    { label: 'yml code', value: 'dbt_project.yml', },
    { label: 'sql code', value: 'models/events/sessions.sql', },
  ]
}>

<TabItem value="dbt_project.yml">

<File name='dbt_project.yml'>

``yaml
name: my_project
version: 1.0.0

...

models:
  +snowflake_warehouse: "EXTRA_SMALL"
  my_project:
    clickstream:
      +snowflake_warehouse: "EXTRA_LARGE"

snapshots:
  +snowflake_warehouse: "EXTRA_LARGE"
```

</File>

</TabItem>
<TabItem value="models/events/sessions.sql">

<File name='models/events/sessions.sql'>

```sql
{{
  config(
    materialized='table',
    snowflake_warehouse='EXTRA_LARGE'
  )
}}
with
aggregated_page_events as (
    select
        session_id,
        min(event_time) as session_start,
        max(event_time) as session_end,
        count(*) as count_page_views
    from {{ source('snowplow', 'event') }}
    group by 1
),
index_sessions as (
    select
        *,
        row_number() over (
            partition by session_id
            order by session_start
        ) as page_view_in_session_index
    from aggregated_page_events
)
select * from index_sessions
```

</File>

</TabItem>
</Tabs>`

Reply:
Sweet! Thanks so much :TIL:

I have made changes and here is the preview: https://deploy-preview-2446--docs-getdbt-com.netlify.app/reference/resource-configs/snowflake-configs